### PR TITLE
Add method to get the registration date to the API

### PR DESCRIPTION
**Pull request by Drc-DEV**
No API method was written when the feature was introduced, there are some use-cases where this data is needed.

### CODE REVIEW

File: src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java [hunk 1]
```diff
@@ -196,6 +196,32 @@ private Long getLastLoginMillis(String playerName) {
...
+     *
+     * @return The timestamp of when the player was registered, or null if the player doesn't exist or is not registered
+     */
+    public Instant getRegistrationTime(String playerName) {
+        Long registrationDate = getRegistrationMillis(playerName);
+        return registrationDate == null ? null : Instant.ofEpochMilli(registrationDate);
+    }
+
+    private Long getRegistrationMillis(String playerName) {
+        if (!isRegistered(playerName.toLowerCase())) {
```
Comment (sgdc3) [inline src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java]: I think that this check can be avoided as we already check if auth != null below

**Comment by ljacqu** [comment]
At this point I think instead of merging this and PR #1978 we should actually do the long-term solution and introduce an ApiUser interface (or whatever we want to call it) that allows to get a _few_ of the interesting values in one API method. I can try to do this maybe this weekend.

**Comment by Drc-DEV** [comment]
> 
> 
> At this point I think instead of merging this and PR #1978 we should actually do the long-term solution and introduce an ApiUser interface (or whatever we want to call it) that allows to get a _few_ of the interesting values in one API method. I can try to do this maybe this weekend.

That would be defintely better, but I'd still add these methods to the current API because they are really essential, and I think that you should still keep the the v3 API for backwards compatibility anyways.

File: src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java (hunk 1 above)
Comment (Drc-DEV) [reply src/main/java/fr/xephi/authme/api/v3/AuthMeApi.java]: Done

**Comment by ljacqu** [comment]
Definitely, I'm talking about adding that one method onto the current API :)

**Comment by sgdc3** [comment]
In the meantime i'll merge this, as we have no eta for the UserApi

**Comment by ljacqu** [comment]
I said this weekend?

**Comment by ljacqu** [comment]
Basically @Drc-DEV count on this being reverted. Sorry for the arbitrary behavior on our side.

**Comment by Drc-DEV** [comment]
> 
> 
> Basically @Drc-DEV count on this being reverted. Sorry for the arbitrary behavior on our side.

Just make it deprecated if you make the UserApi. No need to remove a couple of lines of code.

**Pull request merged**